### DISCUSSION TITLE
LiveBot v1.1.2 alpha release

### DISCUSSION THREAD

**Discussion by SebOuellette** [SebOuellette/LiveBot#127] [Q&A]
## Update Description
Added the ability for the message box to resize based on the text inside it. It no longer keeps the height of 1 row, and will resize up to 6 rows before you need to scroll.

Allowed the user to actually send messages, unlike the previous release. This release has been tested, and all releases in the future will be thouroughly tested as well, to prevent another situation like this. 

<hr /><em>This discussion was created from the release <a href='https://github.com/SebOuellette/LiveBot/releases/tag/v1.1.2-alpha'>LiveBot v1.1.2 alpha release</a>.</em>

**Comment by UnixPlayz** [comment]
yooo

**Comment by Phoenixison** [comment]
Hey this thing is not working for me how can i use it ? Someone help me plss..
![image](https://user-images.githubusercontent.com/73703174/154917117-7e163d0f-331b-4ce0-8928-11478c2c7621.png)
Nothing is showing but i have it in many servers ?

**Comment by rabbanyhmm** [comment]
Try restarting the app, checking bot permissions, or re-inviting the bot.
